app/models/mixins/compliance_mixin.rb

```diff
@@ -34,9 +36,15 @@ def scan_and_check_compliance_queue
     Compliance.scan_and_check_compliance_queue(self)
   end
 
+  # a few of these need to override
+  # name is sometimes {target_class.gsub(/_/,'')}_compliance_check
   def compliance_policies
     target_class = self.class.base_model.name.downcase
     _, plist = MiqPolicy.get_policies_for_target(self, "compliance", "#{target_class}_compliance_check")
     plist
   end
```

Comment:
Instead of overriding the entire method, maybe it's cleaner to create a method justf for the policy name, and then have the subclasses only override _that_.

```ruby
def compliance_policy_name
  "#{self.class.base_model.name.downcase}_compliance_check"
end

  def compliance_policies
    _, plist = MiqPolicy.get_policies_for_target(self, "compliance", compliance_policy_name)
    plist
  end
```